src/EnergyPlus/MixedAir.cc

```diff
@@ -4328,6 +4347,16 @@ namespace MixedAir {
 
 		}
 
+		if ( OAController( OAControllerNum ).CoolCoilFreezeCheck ) {
+			MaxOAFracBySetPoint = min( max( MaxOAFracBySetPoint, 0.0 ), 1.0 );
+			OAController( OAControllerNum ).MaxOAFracBySetPoint = MaxOAFracBySetPoint;
```

Comment:
@mjwitte I ran the same input file using the source code before merge into develop and got non zero values on 4/1 and 4/2. Enclosed is a figure to show the same output variable.
![maxoafraction](https://cloud.githubusercontent.com/assets/5970656/13145654/1f436e7e-d620-11e5-9b6e-d7a0ced3a13e.jpg)

I am going to change a local variable name as l_MaxOAFracBySetPoint. Are you OK with the change?